### DISCUSSION TITLE
Schmalspurbahnen

### DISCUSSION THREAD

**Discussion by C1710** [marhei/TrainCompany-Data#244] [Ideas]
RhB, HSB usw.
Ist mit `neededEquipment`s möglich, aber auch sinnvoll?

Und falls ja, sollten die Tasks eher Ausschreibungen oder Direktvergaben sein? 🤔

**Comment by marhei** [comment]
Würde ich eher nicht machen, zu viele Probleme.

**Comment by f2k1de** [comment]
Aber was mit Breitspurbahnen wenn es nach Spanien oder Osteuropa geht?

**Reply by marhei** [reply]
Bei Spanien würde ich mich erstmal auf die SFS konzentrieren und Osteuropa ist aktuell noch weit weg…
